.circleci/config.yml

```diff
@@ -28,8 +28,31 @@ jobs:
       - run:
           name: Run tests
           command: node common/scripts/install-run-rush.js test --verbose
+  update_npm_dependencies:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run:
+          name: Configure git
+          command: |
+            git config user.email "auto-npm-dep-update@shiftjs.com"
+            git config user.name "autoupdate"
+      - run:
+          name: Update npm dependencies
+          command: node common/scripts/install-run.js rush-update@beta rush-update -x @types/node --repo-owner binaris --repo-name shiftjs
```

Comment:
This is really cool that all the logic is inside one script, just remember to remove the `@beta`.

Reply:
forgot to push :P